schema/deploy/funding_stream_project_status_001_update_hardcoded_values.sql

```diff
@@ -0,0 +1,14 @@
+-- Deploy cif:funding_stream_project_status_001_update_hardcoded_values to pg
+-- requires: tables/project_status
+-- requires: tables/funding_stream
+
+begin;
+
+update cif.funding_stream_project_status set sorting_order = 20 where project_status_id = (select id from cif.project_status where name = 'Agreement Terminated');
+update cif.funding_stream_project_status set sorting_order = 21 where project_status_id = (select id from cif.project_status where name = 'Closed');
```

Comment:
Why 20/21 as the sorting_order values for these records?

Reply:
I chose random values, just made them a little larger in anticipation of a couple additions in the future